site/en/reference/cli/concepts/tcp.md

```diff
@@ -0,0 +1,147 @@
+# Test TCP Functionality With OT CLI
+
+Thread networks offer several `cli tcp` commands for testing peer-to-peer communication
```

Comment:
I didn't notice this in the other UDP review... I don't think "Thread networks offer several `cli tcp` commands", I think OpenThread offers tcp commands for use with a Thread network.  Can you reword to reflect that?  (And in the UDP guide too to match).